crates/astria-conductor/src/executor/mod.rs

```diff
@@ -457,19 +505,36 @@ impl Executor {
         err,
     ))]
     async fn execute_firm(&mut self, block: ReconstructedBlock) -> eyre::Result<()> {
```

Comment:
The same arguments as for `execute_soft` also apply to `execute_firm`. Please see my comments there,